taxi/views.py

```diff
@@ -85,6 +118,24 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
     model = Driver
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(DriverListView, self).get_context_data(**kwargs)
+        username = self.request.GET.get("username")
```

Comment:
Do you need to define class in super when only one parent class has this method? fix everywhere